client/src/plugins/version-info/ReleaseInfo.js

```diff
@@ -48,7 +48,7 @@ export function ReleaseInfo(props) {
           Advance your BPMN diagrams with confidence using this new feature. Hover over a context pad entry to see a preview of the element to create.
         </li>
         <li>
-          <h4>Form Deployment to Camunda 8</h4>
+          <h4>Support for Upcoming Camunda 8 Features - Standalone Form Deployment and Linking</h4>
           You can now deploy forms directly from the forms tab to Camunda 8. To use the deployed form, you can reference it in your BPMN diagram as a linked form.
```

Comment:
```suggestion
You can now deploy forms directly from the forms tab. Reference it from a user task inside a BPMN diagram and Tasklist will pick it up for task completion.
```

Reply:
Or something like: _Supporting upcoming Camunda 8 features, ..._

Reply:
I tried to implement the suggestions in [93017ba](https://github.com/camunda/camunda-modeler/pull/3964/commits/93017ba2343fa38871b46168ec346b81f2c281c8), let me know what you think

Reply:
Great :clap: